TextPet/plugins/exeac-exe3-cht-utf8.tbl

```diff
@@ -0,0 +1,2874 @@
+6DF9=遞
+6DFA=遠
+6DFB=遣
+6DFC=適
+6DFD=遭
+6DFE=遮
+6DFF=遲
+6E00=遵
+6E01=選
+6E02=遼
+6E03=遽
+6E04=避
+6E05=邀
+6E06=邁
+6E07=邂
+6E08=還
+6E09=邊
+6E0A=邏
+6E0B=那
+6E0C=邪
+6E0D=郁
+6E0E=郊
+6E0F=部
+6E10=郵
+6E11=都
+6E12=鄉
+6E13=鄙
+6E14=鄰
+6E15=配
+6E16=酒
+6E17=酥
+6E18=酬
+6E19=酷
+6E1A=酸
+6E1B=醇
+6E1C=醋
+6E1D=醒
+6E1E=醞
+6E1F=醣
+6E20=醫
+6E21=醬
+6E22=釀
+6E23=采
+6E24=釋
+6E25=里
+6E26=釘
+6E27=鈍
+6E28=鈕
+6E29=鈞
+6E2A=鉛
+6E2B=銀
+6E2C=銅
+6E2D=銘
+6E2E=銜
+6E2F=鋭
+6E30=銷
+6E31=鋤
+6E32=鋪
+6E33=鋸
+6E34=錄
+6E35=錢
+6E36=錶
+6E37=鍊
+6E38=鍋
+6E39=鍛
+6E3A=鍬
+6E3B=鍵
+6E3C=鍾
+6E3D=鎧
+6E3E=鎮
+6E3F=鏘
+6E40=鏽
+6E41=鐘
+6E42=鑑
+6E43=鑿
+6E44=門
+6E45=閉
+6E46=閒
+6E47=閘
+6E48=閣
+6E49=閲
+6E4A=閻
+6E4B=闆
+6E4C=闇
+6E4D=闊
+6E4E=闖
+6E4F=關
+6E50=阪
+6E51=阻
+6E52=附
+6E53=陌
+6E54=降
+6E55=限
+6E56=陡
+6E57=院
+6E58=陣
+6E59=陪
+6E5A=陳
+6E5B=陶
+6E5C=隆
+6E5D=隊
+6E5E=階
+6E5F=隔
+6E60=隕
+6E61=隙
+6E62=際
+6E63=障
+6E64=險
+6E65=隸
+6E66=隻
+6E67=雀
+6E68=雇
+6E69=雉
+6E6A=雌
+6E6B=雕
+6E6C=雖
+6E6D=雜
+6E6E=雞
+6E6F=離
+6E70=難
+6E71=雪
+6E72=零
+6E73=需
+6E74=霆
+6E75=霉
+6E76=霸
+6E77=青
+6E78=靜
+6E79=非
+6E7A=靠
+6E7B=靡
+6E7C=革
+6E7D=鞠
+6E7E=韌
+6E7F=頁
+6E80=頂
+6E81=頃
+6E82=項
+6E83=順
+6E84=須
+6E85=預
+6E86=頑
+6E87=頒
+6E88=頗
+6E89=頸
+6E8A=頻
+6E8B=顆
+6E8C=題
+6E8D=額
+6E8E=顏
+6E8F=願
+6E90=顛
+6E91=類
+6E92=顧
+6E93=顫
+6E94=顯
+6E95=颯
+6E96=飄
+6E97=食
+6E98=飢
+6E99=飯
+6E9A=飲
+6E9B=飼
+6E9C=飽
+6E9D=飾
+6E9E=餅
+6E9F=養
+6EA0=餐
+6EA1=餓
+6EA2=餘
+6EA3=餡
+6EA4=館
+6EA5=餵
+6EA6=饋
+6EA7=饒
+6EA8=首
+6EA9=馨
+6EAA=馬
+6EAB=馭
+6EAC=馳
+6EAD=馴
+6EAE=駒
+6EAF=駕
+6EB0=駛
+6EB1=駭
+6EB2=騙
+6EB3=驅
+6EB4=驕
+6EB5=驗
+6EB6=驚
+6EB7=驟
+6EB8=骨
+6EB9=骯
+6EBA=骸
+6EBB=髒
+6EBC=髮
+6EBD=鬆
+6EBE=鬍
+6EBF=鬧
+6EC0=鬱
+6EC1=魁
+6EC2=魄
+6EC3=魅
+6EC4=魚
+6EC5=鮮
+6EC6=鯊
+6EC7=鰍
+6EC8=鳴
+6EC9=鴉
+6ECA=鵡
+6ECB=鶩
+6ECC=鷗
+6ECD=鸚
+6ECE=鹽
+6ECF=鹿
+6ED0=麗
+6ED1=麥
+6ED2=麵
+6ED3=麼
+6ED4=默
+6ED5=點
+6ED6=黨
+6ED7=鼻
+6ED8=齁
+6ED9=齊
+6EDA=齒
+6EDB=龜
+6EDC=砲
+6EDD=燄
+6EDE=穎
+6EDF=拘
+6EE0=咿
+6EE1=呲
+6EE2=噠
+6EE3=嘀
+6EE4=猙
+6EE5=獰
+6EE6=嘣
+6EE7=妖
+6EE8=[SwitchA]
+6EE9=[XboxA]
+6EEA=[PSCross]
+6EEB=[SwitchB]
+6EEC=[XboxB]
+6EED=[PSCircle]
+6EEE=[SwitchY]
+6EEF=[XboxX]
+6EF0=[PSSquare]
+6EF1=[SwitchR]
+6EF2=[XboxRB]
+6EF3=[PSR1]
+6EF4=[SwitchL]
+6EF5=[XboxLB]
+6EF6=[PSRL1]
+6EF7=[SwitchPlus]
+6EF8=[XboxMenu]
+6EF9=[PSOptions]
+6EFA=[SwitchLeft]
+6EFB=[XboxLeft]
+6EFC=[PSLeft]
+6EFD=[SwitchDPad]
+6EFE=[XboxDPad]
+6EFF=[PSDPad]
+6F00=[SwitchRight]
+6F01=[XboxRight]
+6F02=[PSRight]
+6F03=[SwitchLeftRight]
+6F04=[XboxLeftRight]
+6F05=[PSLeftRight]
+6F06=[SwitchDown]
+6F07=[XboxDown]
+6F08=[PSDown]
+6F09=[SwitchUp]
+6F0A=[XboxUp]
+6F0B=[PSUp]
+6F0C=[SwitchZR]
+6F0D=[XboxRT]
+6F0E=[PSR2]
+6F0F=[SwitchZL]
+6F10=[XboxLT]
+6F11=[PSL2]
+6F12=欸
+6F13=錦
+6F14=鯉
+6F15=狐
+6F16=驢
+6F17=轍
+6F18=柿
+6F19=註
+6F1A=炭
+6F1B=剮
+6F1C=喳
+6F1D=眶
+6F1E=絢
+6F1F=踞
+6F20=嘁
+6F21=擒
+6F22=蹟
+6F23=[z]
+6F24=[SwitchUpDown]
+6F25=[XboxUpDown]
+6F26=[PSUpDown]
+6F27=[SwitchDownRight]
+6F28=[XboxDownRight]
+6F29=[PSDownRight]
+6F2A=猿
+6F2B=漁
+6F2C=瀰
+6F2D=矚
+6F2E=托
+6F2F=咯
+6F30=嚼
+6F31=蝕
+6F32=套
+6F33=纏
+6F34=貪
+6F35=湊
+6F36=[$6F36]
+6F37=[$6F37]

```

Comment:
Should be `[V]` and `[S]`; these are not used in `font_cht_05` but do appear in `menu_spr_cht_05`.